account_move_reconcile_helper/models/account_move_line.py

```diff
@@ -0,0 +1,76 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class AccountMoveLine(models.Model):
+
+    _inherit = 'account.move.line'
+
+    partial_reconciliation_in_progress = fields.Boolean(
+        compute='_compute_partial_reconciliation_in_progress')
+    reconcile_line_ids = fields.One2many(
+        compute='_compute_reconciled_lines',
+        comodel_name='account.move.line',
+        string="Reconciled lines")
```

Comment:
I understand why using readonly for related fields. But this is a computed field without an inverse, is this really needed ?